.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
-## [Tracker Story](<URL>)
```

Comment:
Maybe this should be part of our [shared community files](https://github.com/trussworks/.github)?

Reply:
https://github.com/trussworks/.github/pull/8